config/settings.yml

```diff
@@ -28,6 +28,7 @@ rabbitmq:
   password: guest
 
 globus: 
+  enabled: true
```

Comment:
What would you think about disabling by default and then re-enable in QA & stage? That way the "default" behavior is in prod.

Reply:
I think that's 2 shared-config changes instead of one :-)

Reply:
But you're right, it's the right way to do it.

Reply:
Larry Wall does not approve.